scripts/covid_tracking_project/historic_us_data/COVIDTracking_US_StatisticalVariables.mcf

```diff
@@ -0,0 +1,94 @@
+Node: dcid:CumulativeCount_MedicalTest_COVID_19
```

Comment:
Remember that StatVars only need to be checked in once. If these are all the same as States, then we can just reuse.